### PR TITLE
APP-649 Report Integration Versions and Configurator Apps Info

### PR DESCRIPTION
**Pull request by robson-sanchez**
- Health check reports versions for each integration
- Health check reports the configurator app links for icon and configurator app
- Health check reports OK for the integration only if the icon and configurator app links are accessible

### CODE REVIEW

**Review by cmarcondes-daitan — APPROVED** [review]

**Comment by cmarcondes-daitan** [comment]
LGTM, @dnsymphony could you review it please?

File: integration-commons/src/main/java/org/symphonyoss/integration/healthcheck/IntegrationHealthManager.java [hunk 1]
```diff
@@ -16,33 +16,93 @@
...
+  @Autowired
+  private AuthenticationProxy authenticationProxy;
+
+  @Autowired
+  private IntegrationProperties properties;
+
+  /**
+   * Integration settings
+   */
+  private V1Configuration configuration;
```
Comment (dnsymphony) [inline integration-commons/src/main/java/org/symphonyoss/integration/healthcheck/IntegrationHealthManager.java]: Is this a generated class?  If so, it shouldn't be used as a field.

Comment (robson-sanchez) [reply integration-commons/src/main/java/org/symphonyoss/integration/healthcheck/IntegrationHealthManager.java]: This class is not generated by our application, it's imported from the pod-api-client.

Comment (dnsymphony) [reply integration-commons/src/main/java/org/symphonyoss/integration/healthcheck/IntegrationHealthManager.java]: It still should not be used as a field in one of the IB classes.  Copy the data to a IB specific class.

Comment (robson-sanchez) [reply integration-commons/src/main/java/org/symphonyoss/integration/healthcheck/IntegrationHealthManager.java]: FIXED

**Comment by robson-sanchez** [comment]
@dnsymphony please check again

**Review by dnsymphony — APPROVED** [review]

**Pull request merged**